.github/workflows/bug_triager_workflow.yml

```diff
@@ -0,0 +1,33 @@
+name: Bug Triage Workflow
+
+on:
```

Comment:
Try adding pre-commit for some code specific checks 
https://github.com/ansible-collections/cisco.ios/blob/main/.pre-commit-config.yaml

Reply:
sure will do it thanks

Reply:
Completed